Makefile.am

```diff
@@ -251,6 +251,19 @@ AM_CFLAGS = $(OUR_CFLAGS)
 AM_LDFLAGS = $(OUR_LDFLAGS)
 
 # ------------------------------------------------------------------------------
+
+# distros may require that *.so symlinks are always placed in ${libdir}, use
+# this macro to move devel *.so to libdir if needed
+define move-so-to-libdir
+	if test "$(libdir)" != "$(rootlibdir)"; then \
```

Comment:
> These should use $(DESTDIR) as well, no? 

Not really. If $(DESTDIR) was added, the test would become `test "$(DESTDIR)$(libdir) != "$(DESTDIR)$(rootlibdir)"`, then $(DESTDIR) being a common part in both paths can be just skipped.

IIRC `HAVE_SPLIT_USR` is technically only about `/bin|/sbin` symlinks.